### ISSUE TITLE
I left this for a while and started fresh

### ISSUE THREAD

**Issue by suresttexas00** [benbaptist/minecraft-wrapper#50]
.. on a new machine.  I started the wrapper.py file and this is what i get:

C:\Users\JRUSSELL\Desktop\vanilla>python Wrapper.py
Traceback (most recent call last):
  File "C:\Python27\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "Wrapper.py\__main__.py", line 7, in <module>
  File "Wrapper.py\server.py", line 1, in <module>
ImportError: No module named resource

C:\Users\JRUSSELL\Desktop\vanilla>pause
Press any key to continue . . .

**Comment by suresttexas00** [comment]
python version is 2.7.9.  windows 8.1 laptop

**Comment by overjt** [comment]
Download this https://bootstrap.pypa.io/get-pip.py

python get-pip.py

pip install requests pkg_resources pycrypto

**Comment by suresttexas00** [comment]
C:\Users\JRUSSELL\Desktop>pip install requests pkg_resources pycryto
Collecting requests
  Downloading requests-2.5.1-py2.py3-none-any.whl (464kB)
    100% |################################| 466kB 871kB/s ta 0:00:01
Collecting pkg-resources
  Could not find any downloads that satisfy the requirement pkg-resources
  No distributions at all found for pkg-resources

C:\Users\JRUSSELL\Desktop>pip install requests pkg_resources pycryto
Collecting requests
  Using cached requests-2.5.1-py2.py3-none-any.whl
Collecting pkg-resources
  Could not find any downloads that satisfy the requirement pkg-resources
  No distributions at all found for pkg-resources

C:\Users\JRUSSELL\Desktop>

**Comment by suresttexas00** [comment]
and same error as before when i run wrapper.py

**Comment by suresttexas00** [comment]
started from scratch as little and just installed requests(successfully), but pycrypto errors out:

    copying lib\Crypto\Random\__init__.py ->....... [...] ......

    running build_ext

    building 'Crypto.Random.OSRNG.winrandom' extension

    warning: GMP or MPIR library not found; Not building Crypto.PublicKey._fastm
ath.

    error: Microsoft Visual C++ 9.0 is required (Unable to find vcvarsall.bat).
Get it from http://aka.ms/vcpython27

    ----------------------------------------
    Command "C:\Python27\python.exe -c "import setuptools, tokenize;__file__='c:
\\users\\jrussell\\appdata\\local\\temp\\pip-build-yvzl_q\\pycrypto\\setup.py';e
xec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n
'), __file__, 'exec'))" install --record c:\users\jrussell\appdata\local\temp\pi
p-tr8mvx-record\install-record.txt --single-version-externally-managed --compile
" failed with error code 1 in c:\users\jrussell\appdata\local\temp\pip-build-yvz
l_q\pycrypto

C:\Users\JRUSSELL\Desktop>pip list
pip (6.0.6)
requests (2.5.1)
setuptools (7.0)

C:\Users\JRUSSELL\Desktop>

I am sure i have a newer than 9.0 C++ , but I will check...

**Comment by suresttexas00** [comment]
getting the C++ compiler for Python.. WHYYYYY so many dependencies LOL...

**Comment by suresttexas00** [comment]
Still not working.  Same error when starting wrapper.py.  I could install pycryto and requests, but not pkg_resources:

C:\Users\JRUSSELL>pip install pkg_resources
Collecting pkg-resources
  Could not find any downloads that satisfy the requirement pkg-resources
  No distributions at all found for pkg-resources

C:\Users\JRUSSELL>

**Comment by suresttexas00** [comment]
I used an older version of the wrapper (.074 build 61) and it works... so what has changed to cause this  error?

**Comment by suresttexas00** [comment]
works with .7.3 too

**Comment by suresttexas00** [comment]
closing this and opening a different issue #51

**Comment by suresttexas00** [comment]
I removed Python from my system, deleted the C:\python27 folder and re-installed.  Turns out, 2.7.9 python comes with pip.  So i re installed pycrypto and requests.  Pkg_resources is already in the distro.  NOW....  I still can't run anything after build 61 because build 62 imports a module called "resource" in "server.py".. and I don't have any such module.  The name is go generic, I am afraid if I pip search it, there will be a 100 results and I won't know which to use.  Where do I get this module??
